media_utils/video_puppeteer.py

```diff
@@ -148,6 +150,19 @@ def lag(self):
         elapsed_wall_time = clock() - self._start_wall_time
         return elapsed_wall_time - elapsed_current_time
 
+    def check_started(self):
```

Comment:
What do you mean by 'check_started'? Suggestion: rename to something more descriptive like `reset_playback_interval`, say.